frontend/src/app/modules/grids/widgets/add/add.modal.ts

```diff
@@ -5,7 +5,6 @@ import {OpModalLocalsToken} from "app/components/op-modals/op-modal.service";
 import {OpModalLocalsMap} from "app/components/op-modals/op-modal.types";
 import {GridWidgetsService} from "app/modules/grids/widgets/widgets.service";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
-import {AbstractWidgetComponent} from "core-app/modules/grids/widgets/abstract-widget.component";
 import {BannersService} from "core-app/modules/common/enterprise/banners.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/c707e7bf78dbad7cac984139a18fb9d109e530df/frontend/src/app/modules/grids/widgets/add/add.modal.ts#L9"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>